storage/simulation/cluster.go

```diff
@@ -97,6 +100,12 @@ func (c *Cluster) addStore(nodeID proto.NodeID) *Store {
 	s := n.addNewStore()
 	storeID, _ := s.getIDs()
 	c.stores[storeID] = s
+
+	// Save a sorted array of store IDs to avoid having to calculate them
+	// multiple times.
+	c.storeIDs = append(c.storeIDs, storeID)
+	sort.Sort(proto.StoreIDSlice(c.storeIDs))
```

Comment:
`sort.Search()` can be used to insert into a sorted array but it's a bit of a pain to use (because go can't express an "insert into sorted array" function that is both efficient and generic).